### PR TITLE
Revise Tutorial.md

### PR DESCRIPTION
**Pull request by hyunsik-yoon**
This revises Tutorial.md.

ONE-vscode-DCO-1.0-Signed-off-by: Hyun Sik Yoon <hyunsik.yoon.1024@gmail.com>

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -80,35 +80,40 @@ The folder for ONE-vscode needs to have the following files:
 
 ## Install the toolchain
 
-You need the ONE toolchain. Let's install the ONE toolchain. (You should install the in-house backend to follow this.)
+A toolchain contains [ONE tools](https://github.com/Samsung/ONE/tree/master/compiler/one-cmds) and other backend-specific tools. Developers or vendors of backend can distribute backend-specific toolchain and put installation feature inside their backend extension.
```
Comment (llFreetimell) [inline docs/Tutorial.md:83]:
```suggestion
A toolchain contains [ONE command tools](https://github.com/Samsung/ONE/tree/master/compiler/one-cmds) and other backend-specific tools. Developers or vendors of backend can distribute backend-specific toolchain and put installation feature inside their backend extension.
```

File: docs/Tutorial.md [hunk 2]
```diff
@@ -80,35 +80,40 @@ The folder for ONE-vscode needs to have the following files:
...
+To decide which toolchain will be used by default, you need to specify the default toolchain. Default toolchain refers to which toolchain you will use to run ONE related tools. To specify a default toolchain, if you mouse up the toolchain item you want to use in Toolchain view, :heavy_check_mark: icon appears. Click this icon to specify it as default toolchain. If toolchain is specified as default toolchain, the icon changes.
 
 | Toolchain | Default Toolchain |
 |-----------|-------------------|
 | <img src="https://user-images.githubusercontent.com/7223627/177962615-71e64f52-e684-432d-a053-07896a20a479.png" width=300 /> | <img src="https://user-images.githubusercontent.com/7223627/177963453-d22998a0-be21-4531-a70b-9057a204e7eb.png" width=300 /> |
 
 ## Prepare a NN model and a config file
 
-You need a NN model and a config file to run compiling. Now you can get one from [res/modelDir/truediv](../res/modelDir/truediv/). Add it to your workspace. Now you can see a model and a config.
+Now, let's try to copile a model (tflite, Tensorflow, onnx).
```
Comment (llFreetimell) [inline docs/Tutorial.md:99]:
```suggestion
Now, let's try to compile a model (tflite, Tensorflow, onnx).
```

File: docs/Tutorial.md [hunk 3]
```diff
@@ -80,35 +80,40 @@ The folder for ONE-vscode needs to have the following files:
...
+
+Just like you might write _Makefile_ for C progream, you need ONE `*.cfg` file to compile a model. ONE cfg file is a file that describes steps and compilation options.
```
Comment (llFreetimell) [inline docs/Tutorial.md:101]:
```suggestion
Just like you might write _Makefile_ for C progream, you need ONE `*.cfg` file to compile a model. ONE cfg file is a file that describes steps and options of compilation.
```

File: docs/Tutorial.md [hunk 4]
```diff
@@ -80,35 +80,40 @@ The folder for ONE-vscode needs to have the following files:
...
+You can also edit the cfg file with GUI cfg editor.
 
-## Compile models
+<img src="one-explorer-cfg-editor.gif"/>
 
-Last, you can compile the model by pushing Run button.
+## Compile a model
 
-Depends on backend, you may be able to infer or profile a model by 'running config'.
+Now, you can compile the model by clicking `▶` icon on a cfg file. Below cfg file in ONE explorer, you can find files during compilation, e.g., various `*.circle` file and the final target file compiled for your backend.
```
Comment (llFreetimell) [inline docs/Tutorial.md:110]:
```suggestion
Now, you can compile the model by clicking `▶` icon on a cfg file. Below cfg file in ONE explorer, you can find generated files by compilation, e.g., various `*.circle` file and the final target file compiled for your backend.
```

File: docs/Tutorial.md [hunk 5]
```diff
@@ -80,35 +80,40 @@ The folder for ONE-vscode needs to have the following files:
...
+Now, let's try to copile a model (tflite, Tensorflow, onnx).
+
+Just like you might write _Makefile_ for C progream, you need ONE `*.cfg` file to compile a model. ONE cfg file is a file that describes steps and compilation options.
+One example you can get is in [res/modelDir/truediv](../res/modelDir/truediv/). You may want to copy and modify the cfg file to your workspace. After modifying the cfg file to have your model file as input model, you will see the cfg file below your model file in ONE explorer.
 
-<img src="https://user-images.githubusercontent.com/10216715/174798969-eee44fea-bd71-4e6a-8e2c-9e1de37ad74a.gif" width=600 />
+You can also edit the cfg file with GUI cfg editor.
 
-## Compile models
+<img src="one-explorer-cfg-editor.gif"/>
```
Comment (hyunsik-yoon) [inline docs/Tutorial.md:106]: @llFreetimell I will make this gif again as we discussed offline.

File: docs/Tutorial.md [hunk 6]
```diff
@@ -51,64 +50,67 @@ ONE-vscode has 6 screen components.
...
 
-<img src="https://user-images.githubusercontent.com/7223627/174947559-71ab213c-68c4-4700-bcec-e55f6a2203bf.gif" width=600 />
+To install a backend-specific toolchain, you first need to install the backend extension. Then, click `+` icon on the Toolchain view. You can also easily remove installed toolchain by clicking `X` icon in the Toolchain view.
 
 ## Set default toolchain
 
-To compile model using ONE toolchain, you must specify the default toolchain. Default toolchain refers to which toolchain you will use to run ONE related tools. To specify a default toolchain, if you mouse up the toolchain item you want to use in Toolchain view, :heavy_check_mark: icon appears. Click this icon to specify it as default toolchain. If toolchain is specified as default toolchain, the icon changes.
+You can install multiple instances of toolchains. For example, some backend supports multiple versions of toolchain to be installed. Or you might installed more than one backend extensions installed and installed one or more toolchain per backend extension.
+
```
Comment (jyoungyun) [inline docs/Tutorial.md:87]: The purpose of writing this tutorial was to tell user that default toolchain setting is essential to execute config. Could you add the contents about this?
> To compile model using ONE toolchain, you must specify the default toolchain.

File: docs/Tutorial.md [hunk 7]
```diff
@@ -51,64 +50,67 @@ ONE-vscode has 6 screen components.
...
+To decide which toolchain will be used by default, you need to specify the default toolchain. Default toolchain refers to which toolchain you will use to run ONE related tools. To specify a default toolchain, if you mouse up the toolchain item you want to use in the Toolchain view, :heavy_check_mark: icon appears. Click this icon to specify it as default toolchain. If toolchain is specified as default toolchain, the icon changes.
 
 | Toolchain | Default Toolchain |
 |-----------|-------------------|
 | <img src="https://user-images.githubusercontent.com/7223627/177962615-71e64f52-e684-432d-a053-07896a20a479.png" width=300 /> | <img src="https://user-images.githubusercontent.com/7223627/177963453-d22998a0-be21-4531-a70b-9057a204e7eb.png" width=300 /> |
```
Comment (jyoungyun) [inline docs/Tutorial.md:92]:
```suggestion
To decide which toolchain will be used by default, you need to specify the default toolchain. Default toolchain refers to which toolchain you will use to run ONE related tools. To specify a default toolchain, if you mouse up the toolchain item you want to use in the Toolchain view, :heavy_check_mark: icon appears. Click this icon to specify it as default toolchain.
```

File: docs/Tutorial.md (hunk 6 above)
Comment (hyunsik-yoon) [reply docs/Tutorial.md:87]: I think current statement at L88

> To decide which toolchain will be used by default, you need to specify the default toolchain.

is same with the statement you'd like to add. 

Do you want to replace current statement with the one you wrote like the following?

`To decide which toolchain will be used by default, you need to specify the default toolchain.`
to 
`To compile model using ONE toolchain, you must specify the default toolchain.`
?

Comment (jyoungyun) [reply docs/Tutorial.md:87]: `To execute config file in ONE explorer, you must specify the default toolchain.`
How about it?

**Comment by hyunsik-yoon** [comment]
PTAL

**Review by jyoungyun — APPROVED** [review]
LGTM Thank you.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**